.storybook/manager-head.html

```diff
@@ -0,0 +1 @@
+<script>document.title = "UI component library - commercetools";</script>
```

Comment:
I would call it UI Kit to avoid introducing many terms which has shown to be confusing.